woocommerce/woocommerce-ios
# [CPP Onboarding] Implement Stripe account restricted due to overdue requirements

Part of #4611 

This adds an specific error when the Stripe account is restricted due to overdue requirements.

<img src="https://user-images.githubusercontent.com/2722505/128898136-259b50f9-ca45-4238-9a83-01959d7ed6f6.png" width="350"/>

## How to test
In order to test this in real conditions, it is necessary to have a store associated to an Stripe account that is has overdue requirements, or...
make `CardPresentPaymentsOnboardingUseCase.checkOnboardingState` return `stripeAccountOverdueRequirement`

1. Navigate to Settings > In-Person Payments
2. Notice the error screen.

Update release notes:
